### PR TITLE
Manage Creation of UI configuration ConfigMaps by HCO

### PR DESCRIPTION
**Pull request by orenc1**
Currently, when kubevirt-hco is installed, the console-plugin configuration configmaps are created only when a cluster-admin opens the Virtualization tab for the first time. At that point two ConfigMaps are created in the installation namespace:
- kubevirt-user-settings
- kubevirt-ui-features

And also their RBACs to be reachable by regular users are created. Without login by an admin, the config maps are missing, and regular users cannot use the UI at that point.

This PR adds logic to create those configmaps and associated RBACs, if they are not already exist.
Also, remove the ConfigMaps from their previous location at the default namespace, used in v1.12.0 and below.

**What this PR does / why we need it**:

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly

**Jira Ticket**:

```jira-ticket
https://issues.redhat.com/browse/CNV-49492
```

**Release note**:

```release-note
Manage Creation of UI configuration ConfigMaps by HCO
```

### CODE REVIEW

File: assets/upgradePatches.json [hunk 1]
```diff
@@ -162,6 +162,18 @@
...
+      "semverRange": "<1.14.0",
+      "groupVersionKind": {
+        "group": "",
+        "version": "v1",
+        "kind": "ConfigMap"
+      },
+      "objectKey": {
+        "name": "kubevirt-ui-features",
+        "namespace": "default"
+      }
```
Comment (nunnatsa) [inline assets/upgradePatches.json:176]: why do we need this?

**Review by nunnatsa — COMMENTED** [review]
One inline question

File: assets/upgradePatches.json (hunk 1 above)
Comment (orenc1) [reply assets/upgradePatches.json:176]: on the other hand, I see I forgot to add the removal of the second CM (`kubevirt-user-settings`) and their Roles and RoleBindings. 6 objects in total. Do you think we should remove all of them? or just leave them as is?

Comment (nunnatsa) [reply assets/upgradePatches.json:176]: oh, I missed the `objectsToBeRemoved` - it was out  side of my screen... So yes, please remove everything.

**Review by nunnatsa — APPROVED** [review]
/approve

**Pull request merged**